index.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+var _ = require('lodash');
+
+var predicates = require('./predicates');
+
+/**
+ * A plugin for Irken applications that exposes keyboard shortcut functionality.
+ * @param  {object}   app  Instance of Irken application
+ * @param  {object}  opts Options to be passed to Iggins plugin.
+ * @param  {Function} done Noop function
+ * @return {undefined}        No return value.
+ * @example
+ *
+ * // Relies on having a Irken application.
+ * var Irken = require('irken');
+ * var app = new Irken();
+ *
+ * var plugins = [
+ *   {
+ *     register: require('iggins')
+ *   }
+ * ];
+ *
+ * // Is registered on an Irken applicaion as a plugin.
+ * // This exposes the keypress functionality.
+ * app.register(plugins, function(err){
+ *  console.log('registered', err, app);
+ *  app.render(function(err){
+ *    console.log('rendered', err);
+ *  });
+ * });
+ *
+ * //For debugging
+ * window.app = app;
+ *
+ * //Verify that the keypress function is exposed on the Irken application.
+ * console.log(typeof app.keypress);
```

Comment:
registration is done async, this won't work or be available here, only inside the register callback